### DISCUSSION TITLE
Welcome to ONEST Community Discussions!

### DISCUSSION THREAD

**Discussion by surendrasinghs** [ONEST-Network/community#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by datt2409** [comment]
how can work onest network and how can i use api
